### DISCUSSION TITLE
Should I commit files that changed after running ./gradle jApiCmp

### DISCUSSION THREAD

**Discussion by hongshuai1994** [git-ecosystem/git-credential-manager#1654] [Q&A]
I am trying to submit a PR, but some Checks have not passed. Check the Checks error content. The prompts are as follows.
![image](https://github.com/git-ecosystem/git-credential-manager/assets/80237674/0f807d52-1bc6-4703-9ae7-75b4f5a12348)
I ran .`/gradlew jApiCmp`, but file modifications occurred. Should I submit these files to the PR?
![image](https://github.com/git-ecosystem/git-credential-manager/assets/80237674/acedd7a4-2dce-4b1c-a28a-032f29333e8f)

**Comment by hongshuai1994** [comment]
Sorry, I submitted it to the wrong repository.

**Reply by ghost** [reply]
Y que vergas isiste

**Reply by ghost** [reply]
Eres un hijo de puta por qué noo mejor te botas a chingar a tu puta madre puñetas
